src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/action/CreateIssueAction.kt

```diff
@@ -31,11 +31,11 @@ class CreateIssueAction(
                 .waitForDialog()
                 .showAllFields()
                 .selectProject(project.name)
-                .selectIssueType(
+                .selectIssueType { issueTypes ->
                     seededRandom.pick(
-                        issueCreateDialog.getIssueTypes().filter { it != "Epic" }
+                        issueTypes.filter { it != "Epic" }
```

Comment:
FYI the filtering is no longer necessary. It was done to avoid filling in a mandatory `Epic Link` field a loooong time ago.
Now we fill in all required fields automatically, a few lines below: `issueCreateDialog.fillRequiredFields()`.

Reply:
OK, removing then.